### ISSUE TITLE
Error when download the code

### ISSUE THREAD

**Issue by starry-y** [microsoft/TAP#28]
`failed to perform copy command due to error: cannot start job due to error: cannot list files due to reason read tcp 192.168.0.178:51805->52.239.247.100:443: read: connection reset by peer.`

**Comment by zyang-ur** [comment]
It seems to be working on my side.
Could you please kindly share more background/error information? Thanks

**Comment by starry-y** [comment]
Thanks for your reply !
Here is my details.

I got error when I used AzCopy to download [TextVQA/Caps/STVQA Data (~62G)].
I did this command '`path/to/azcopy copy https://tapvqacaption.blob.core.windows.net/data/data <local_path>/data --recursive`' . 
But the download process stops in the half.

I can download `model checkpoints` using AzCopy, but not TextVQA/Caps/STVQA Data.
The error log is `23.8 %, 17252 Done, 72746 Failed, 2 Pending, 0 Skipped, 90000 Total (scanning...), 
failed to perform copy command due to error: cannot start job due to error: cannot list files due to reason read tcp 172.20.10.2:51974->52.239.247.100:443: read: connection reset by peer.``

Thanks for your attention, and I wonder is there any other way to download the data ?
